cypress/support/pages/yves/company/user/manage/repositories/suite-manage-company-role-user-repository.ts

```diff
@@ -0,0 +1,10 @@
+import { injectable } from 'inversify';
+import { ManageCompanyRoleUserRepository } from '../manage-company-role-user-repository';
+
+@injectable()
+export class SuiteManageCompanyRoleUserRepository implements ManageCompanyRoleUserRepository {
+  getFirstUserUnassignButton = (): Cypress.Chainable =>
+    cy.get('body').find('.role-user-table tr:first-child a:contains("Unassign")');
+  getFirstUserAssignButton = (): Cypress.Chainable =>
+    cy.get('body').find('.role-user-table tr:first-child a:contains("Assign")');
```

Comment:
let's change the logic from 'contains the text' to 'a contains in href' for all repositories.

- `cy.get('.role-user-table tr:first-child a[href*="user/unassign"]');`
- `cy.get('.role-user-table tr:first-child a[href*="user/assign"]');`